src/Controls/src/Core/Shell/BaseShellItem.cs

```diff
@@ -225,6 +229,11 @@ static void OnIconChanged(BindableObject bindable, object oldValue, object newVa
 			shellItem.FlyoutIcon = (ImageSource)newValue;
 		}
 
+		static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			Shell.SetFlyoutItemIsVisible(bindable, (bool)newValue);
```

Comment:
Do we need to worry about keeping this synced up if the reverse happens? 

So, if someone has

`<FlyoutItem Shell.FlyoutItemIsVisible="true"`

do we need to set the new BP to match the attached property?

Reply:
I can fix that, I need to figure out how to install the workloads for .NET 9.

Reply:
if you're on windows .NET9 shipped with the latest VS PReview
so it'll just install everything you need

Reply:
I only have Windows on my work computer. But I got it working on my Mac too with some support from Gerald. There was some strange versions of .NET workloads installed. I had to remove .NET and intall it again from scratch. 

But now I had added code so the properties will match. I also added a test page so it will be easier to test it.